W-13840863-EntitiesOverview-LimitedGA-SY (#255)
mulesoft/docs-monitoring: modules/ROOT/pages/entities-overview.adoc

== Access the Entities Overview Page

include::partial$include-nav-steps-no-permissions.adoc[]

include::partial$include-nav-entities.adoc[]

The *Entities Overview* page appears, showing a summary of data for entities in the current business group and environment.

== Filtering Data in the Entity Overview Page

By default, the *Entity Overview* page shows data for entities in your current business group and environment. You can change or remove these filters to see data for entities that are in different environments or business groups by clicking the *X* next to each option.

Quick filters, such as `Top 10 entities with highest error volume`, offer you common scenarios at a glance, but you can also use attributes and metrics to filter content that appears in the *Entity Overview* page. You can combine diffrent filters to capture specific combinations of data. 

You can filter entities using the following attributes: 

* Environment 
* Entity name 
* Entity type 
* Status
* Response time 
* Requests
* Errors 
* Client App (for APIs)
* Locations (for APIs)
* Org ID 

You can filter entities with the following metrics:

> 

Author: // need to add

* Environment 
* Entity name 
* Entity type 
* Status
* Response time 
* Requests
* Errors 
* Client App (for APIs)
* Locations (for APIs)
* Org ID 

You can curate your search results for entity data using the following metrics:

tbd

=== Applying Search Parameters

include::partial$include-nav-steps-no-permissions.adoc[]

include::partial$include-nav-entities.adoc[]

. In the search bar, enter an attribute, element, business group, or environment you want to use to filter your data.

=== Example of Applied Search Parameters

tbd

== Filtering Data using the Time Period Selector

You can display the data points collected for the last given period of time (such as the last 5 or 30 minutes) or over a given date and time range. 

Organizations with a Gold, Platinum, or Starter subscription can view up to 30 days of data. Organizations with a Titanium or Advanced subscription can view up to one year of data.

=== Setting the Time Period to Display

Use the drop-down in the calendar icon to select the time period to display in a graph.

Your setting applies across all charts in the entity summary. 

=== Changing the Time Period to Display

> You can display an earlier or later time period in your chart, and you can zoom in or out to cover a shorter or longer period of time.

Reviewer: right now we don't support zoom-in/zoom-out functionality on the date picker.

Author: ```suggestion
You can display an earlier or later time period in your chart to cover a shorter or longer period of time.
```

> [cols="1,4"]
> |===
> | image:zoom-earlier-later.png[Zoom Out, Show Earlier and Later] a|
> To display earlier or later data, you can use the less-than (&lt;) and greater-than (&gt;) icons.
> 
> To zoom out and display data over a longer time period, you use the magnifying glass icon located between the less-than and greater-than icons
> |===

Author: ```suggestion
```

// steal Setting Data Display Options in a Dashboard and Setting the Time Period Display from dashboards-using.adoc but change the titles.

// see if Changing the Time Period to Display still applies.

// steal Refreshing Data

== Understanding Summary and Performance Trends for Entities

> The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment. 

Author: ```suggestion
The *Entities Overview* page shows charts that detail entity performance. By default, the charts show data for entities in your current business group and environment. When you modify your search settings, the data in the charts changes accordingly.
```

=== Summary Charts

Total Entities::

> The total number of entities according to your current filter conditions.

Author: ```suggestion
The total number of entities according to your current search conditions.
```

Total Requests::

> The sum of inbound and outbound requests according to your current filter conditions.

Author: ```suggestion
The sum of inbound and outbound requests according to your current search conditions.
```

Average Latency::

> [I don't know] , calculated in milliseconds. 

Author: @vimil-saju Can you help with a definition here?

Average Error Rate::

> The percentage of requests that result in errors according to your current filter conditions.

Author: ```suggestion
The percentage of requests that result in errors according to your current search conditions.
```

> Active Alerts::

Reviewer: we don't show Active Alerts for initial launch. We have Active Entities card instead.

Author: ```suggestion
Active Entities::
```

> The total number of alerts configured for entities according to your current filter conditions.

Author: ```suggestion
The total number of active entities according to your current search conditions.
```

=== Performance Trends Charts

Request Latency::

> [I don't know], calculated in milliseconds. 

Author: @vimil-saju definition here too?

Author: @sathishpaulleo cc

Error Volume::

> The total number of failed inbound and outbound requests to entities according to your current filter conditions.

Author: ```suggestion
The total number of failed inbound and outbound requests to entities according to your current search conditions.
```

Request Volume:: 

> The total number of inbound and outbound requests to entities according to your current filter conditions.

Author: ```suggestion
The total number of inbound and outbound requests to entities according to your current search conditions.
```

Organizations with a Gold, Platinum, or Starter subscription can view up to 30 days of data. Organizations with a Titanium or Advanced subscription can view up to one year of data.

[NOTE]: If you do not see data in any of the cards, charts, or tables, ensure that you have chosen the appropriate date-time range in the date picker.

//Organizations with a Titanium or Advanced Tier also have the ability to automatically refresh the *Entities Overview* page. The minimum duration for which auto-refresh can be enabled is 5 minutes. Manual refreshing of data is available to users of all tiers.
//From tech review, but seems redundant. 

=== Setting the Time Period to Display

Use the drop-down in the calendar icon to select the time period to display in a graph.

Your setting applies across all charts in the entity summary. 

=== Changing the Time Period to Display

You can display an earlier or later time period in your chart to cover a shorter or longer period of time.

// steal Setting Data Display Options in a Dashboard and Setting the Time Period Display from dashboards-using.adoc but change the titles.

// see if Changing the Time Period to Display still applies.

> // steal Refreshing Data

Author: ```suggestion
```
